src/mono/sample/wasm/browser/main.mjs

```diff
@@ -1,21 +1,25 @@
 import createDotnetRuntime from './dotnet.js'
 
-const { MONO, BINDING, Module, RuntimeBuildInfo } = await createDotnetRuntime((api) => ({
-    disableDotnet6Compatibility: true,
-    configSrc: "./mono-config.json",
-    onAbort: () => {
-        wasm_exit(1);
-    },
-}));
-
 function wasm_exit(exit_code) {
     console.log(`WASM EXIT ${exit_code}`);
 }
 
-const testMeaning = BINDING.bind_static_method("[Wasm.Browser.Sample] Sample.Test:TestMeaning");
-const ret = testMeaning();
-document.getElementById("out").innerHTML = `${ret} as computed on dotnet ver ${RuntimeBuildInfo.ProductVersion}`;
+try {
+    const { MONO, BINDING, Module, RuntimeBuildInfo } = await createDotnetRuntime((api) => ({
+        disableDotnet6Compatibility: true,
+        configSrc: "./mono-config.json",
+        onAbort: (err) => {
```

Comment:
Is there any difference between `try-catch` and `onAbort` callback? 
With the await syntax, the `try-catch` looks much more natural.

Reply:
This is emscripten event. We could possibly install our own handler, when there is no user handler and propagate the error to the top level promise.

Reply:
this is done in https://github.com/dotnet/runtime/pull/62587